src/builder/create_embed.rs

```diff
@@ -269,6 +269,12 @@ impl From<EmbedAuthor> for CreateEmbedAuthor {
     }
 }
 
+impl From<User> for CreateEmbedAuthor {
+    fn from(user: User) -> Self {
+        Self::new(&user.name).icon_url(user.face())
```

Comment:
You should pass user.name is as-is, to avoid cloning.
```suggestion
        Self::new(user.name).icon_url(user.face())
```

Reply:
actually no this doesn't work because
```
error[E0382]: borrow of partially moved value: `user`
   --> src/builder/create_embed.rs:274:39
    |
274 |         Self::new(user.name).icon_url(user.face())
    |                   ---------           ^^^^ value borrowed here after partial move
    |                   |
    |                   value partially moved here
    |
    = note: partial move occurs because `user.name` has type `std::string::String`, which does not implement the `Copy` trait
   ```

Reply:
Call user.face before passing in the name then, I should have spotted that when making the suggestion lol.

Reply:
Yeah also bad call from me to just commit that. But CreateEmbedAuthor::new uses user.name as argument so I guess I should just make it without the builder function via From<EmbedAuthor> but wouldn't that also clone?

Reply:
No. You can call user.face() and store the resulting String in a variable, then pass user.name into the builder, then pass the String into the builder.

Reply:
oh right